doc/colvars-refman-main.tex

```diff
@@ -2672,10 +2661,11 @@ \subsection{LAMMPS keywords}
     \texttt{colvar}}{%
     Output file for the time correlation function}{%
     UNIX filename}{%
-    \texttt{$<$name$>$.corrfunc.dat}}{%
+    \outputName\texttt{.$<$name$>$.corrfunc.dat}}{%
```

Comment:
I'd leave the `.dat` extension, because it is an output file written all at once at the end of the simulation or restart cycle (more precisely, when `write_output_files()` is called), as opposed to incrementally during a run, which is a `.traj` file.

I'll update the `run_tests` script, though.